config/charts/inferencepool/templates/epp-servicemonitor.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.inferenceExtension.monitoring.prometheus.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "gateway-api-inference-extension.name" . }}-monitor
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gateway-api-inference-extension.labels" . | nindent 4 }}
+spec:
+  endpoints:
+  - interval: {{ .Values.inferenceExtension.monitoring.interval }}
+    port: {{ .Values.inferenceExtension.monitoring.port }}
+    path: {{ .Values.inferenceExtension.monitoring.path }}
+    authorization:
```

Comment:
If ServiceMonitor is namespace-scoped, does the secret need to reside in the same namespace of the CR?

Reply:
AFAIK - but I've never run with the secret in another ns so not 100% sure - definitely I'd say best practice though.

Reply:
(namespace is included in the secret template)

Reply:
AFAICT there is no option to add namespace in that authorization.credentials section.